### PR TITLE
Add basic readme with file layout. Connect to travis

### PR DESCRIPTION
**Pull request by aleffert**

### CODE REVIEW

**Comment by aleffert** [comment]
please review @benpatterson @e0d cc @feanil @jzoldak

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+    ├── <...>                           # A folder for every project!
+    ├── shared                          # Code shared between each project
+    │   ├── main
+    │   │   ├── groovy                  # support classes
+    │   │   └── resources
+    │   │       └── idea.gdsl           # IDE support for IDEA
+    │   └── test
+    │       └── groovy                  # specs
+    └── build.gradle                    # build file
+    └── hosts
```
Comment (benpatterson) [inline README.md]: I could also see this as `seeds` which could arguably be run anywhere and don't tie us to specific URLs.

**Comment by aleffert** [comment]
Created https://openedx.atlassian.net/browse/DEVOPS-4243 for the travis side

**Comment by benpatterson** [comment]
@aleffert I just added this repo to travis. Should be running builds on future commits.

File: README.md (hunk 1 above)
Comment (aleffert) [inline README.md]: Is that just a matter of leaving ".edx.org" off the names of the hosts?

Comment (aleffert) [inline README.md]: I guess ``seeds`` would also be a fine folder name even if the files just end up being hostnames anyway. Will change.

**Comment by aleffert** [comment]
@benpatterson anything else before I merge?

**Comment by benpatterson** [comment]
:+1:

**Pull request merged**